blockchain/types/transaction.go

```diff
@@ -181,6 +190,15 @@ func (tx *Transaction) EncodeRLP(w io.Writer) error {
 	return rlp.Encode(w, serializer)
 }
 
+// MarshalBinary returns the canonical encoding of the transaction.
+// For legacy transactions, it returns the RLP encoding. For EIP-2718 typed
```

Comment:
Klaytn has other types as well as EIP-2718
```suggestion
// For legacy transactions, it returns the RLP encoding. For typed
```